core/src/main/java/org/infinispan/manager/CacheManagerInfo.java

```diff
@@ -127,6 +132,15 @@ public boolean isSiteCoordinator() {
       return transport != null && transport.isSiteCoordinator();
    }
 
+   public boolean isRebalancingEnabled() {
+      try {
+         return localTopologyManager.isRebalancingEnabled();
+      } catch (Exception e) {
+         // Do nothing
+      }
+      return false;
+   }
+
```

Comment:
I don't think this is right. If the rebalancing status cannot be retrieved it should not be set to `false` as this may not be the case, instead it should be null. That way the console (or other clients) can indicate that the status is actually unknown if `null` is returned.

So if you change this to be like your original implementation in the `ContainerResource`, where we return `Boolean` with null if a `Exception` occurs.

Reply:
According to the code, I really don't know when this is supposed to raise an exception and become a not redeable value